src/frontend/test/features/featurefiles/e2e/MPO_ViewPIA.feature

```diff
@@ -102,6 +102,7 @@ Scenario Outline: As <role> user, I am able to submit a valid form with all fiel
     Then I click on "Add a role" button 
     And I enter "role"
     Then I click on "Add" button
+    Then I click on "Edit review" button
```

Comment:
It shouldn't be showing the edit review button by default, it should just be editable. We'll have to fix that.

Reply:
Right, it should not be showing it by default but on the code base auto saved should not make non editable section(view section).